scripts/README.md

```diff
@@ -0,0 +1,10 @@
+These are scripts used by the CI for deployment, etc.
+
+
+### cdn_deploy.js
+
+Deploys code and source maps to the CDN.
+Requires aws-sdk, nodegit and minimist:
+```
+# npm install --no-save aws-sdk nodegit minimist
```

Comment:
I was hesitant to add them into the package since 99% of the time this script won't be used, but if it's okay I'm happy to add them in to dev dependencies